app/models.py

```diff
@@ -1666,7 +1666,9 @@ class Brief(db.Model):
                            default=utcnow)
     updated_at = db.Column(DateTime, index=True, nullable=False,
                            default=utcnow, onupdate=utcnow)
-    published_at = db.Column(DateTime, index=True, nullable=True)
+    _published_at = db.Column("published_at", DateTime, index=True, nullable=True)
```

Comment:
why the `_` ?

Reply:
to intercept the setting of this value in tests which access the db model directly rather than via API.

Reply:
https://github.com/AusDTO/dto-digitalmarketplace-api/pull/375/files#diff-413e9aa8961126d20e2d90befa13e0f1R1908